graphql/tests/patch/results.json

```diff
@@ -134,7 +134,7 @@
         "data": {
           "patch": {
             "project": {
-              "tasks": ["compile", "test", "lint", "coverage", "e2e_test"],
+              "tasks": [],
```

Comment:
We can remove this field from the associated test query.

Reply:
and remove it from the results.json

Reply:
the test will fail until the field is removed from the schema since the field is non-nullable